### DISCUSSION TITLE
Make realtime goal conversions (last 30 min) clickable

### DISCUSSION THREAD

**Discussion by davidkarpik** [plausible/analytics#520] [Anything Else]
It would be great, if one could click below Goal Conversions (last 30 min) to see the conversions that happened in the last 30 mins. Currently it works in any time period but not in realtime period.

**Update:** Goal conversions in the realtime view are now clickable

**Comment by bencresty** [comment]
I was missing that too while developing. A page refresh in Plausible on any other time period works pretty quick, so there doesn't seem to be a hurdle to make it clickable too in realtime mode.

**Comment by Olakeen** [comment]
Up this.

Goal conversions in realtime are not clickable. A possible way to "show them" is to add a filter. Please make it clickable 👍 
![2022-06-16 182132](https://user-images.githubusercontent.com/26546237/174118943-51b6d620-d846-4b54-9924-782786a04668.jpg)

**Comment by metmarkosaric** [comment]
**Update:** Goal conversions in the realtime view are now clickable

**Comment by davidkarpik** [comment]
Unfortunately, they aren't clickable anymore, in real time or any other time frame. Could you please bring this function back?

**Reply by metmarkosaric** [reply]
hi @davidkarpik! we haven't changed anything about goals. goals are still clickable. we did just launch a dedicated tab for "Properties". "Properties" tab provides a list view of all the custom properties you're collecting, both at the custom event and the pageview levels. but to see all properties in the list, you must first explicitly add them in the "Properties" section of the site settings, similar to how it works for custom events themselves. you can add all the existing properties with a single click. this feature also gives you the flexibility to choose which properties you want to display (or not) in the dashboard list, similar to the control you have over custom events

**Reply by davidkarpik** [reply]
Hey Marko, thanks for the prompt answer. To clarify, my workflow is to click on the little arrow showing up next to an outbound link, so I could see which outbound links are the most popular on any given page (the link urls are not descriptive in my case). The arrow now disappeared. This might have something to do with the new properties functionality, or someone just accidentally removed it, when properties were set up?
<img width="268" alt="Screenshot 2023-08-07 at 17 37 55" src="https://github.com/plausible/analytics/assets/75309901/dee4bf35-db21-42e2-9bcb-4931db466113">

**Reply by metmarkosaric** [reply]
ahhh i understand. that's accidental. we'll return those arrow asap. thanks for reporting @davidkarpik!

**Reply by metmarkosaric** [reply]
the arrows are now back! thanks @davidkarpik!

**Reply by davidkarpik** [reply]
Thanks!
Is there a way to bring the arrows back to the modal windows? I mean the ones that open up when you click on "[ ] DETAILS".

**Reply by metmarkosaric** [reply]
that one would be more complicated. we've never had it there in the pages reports either. i've put it on our list of things to consider so we'll see what can be done

**Reply by davidkarpik** [reply]
I see, thanks for the clarification.
